src/vs/workbench/contrib/scm/browser/util.ts

```diff
@@ -78,6 +78,16 @@ export function collectContextMenuActions(menu: IMenu): IAction[] {
 	return actions;
 }
 
+export function compareInlineMenuWithInlineActionBar(menu: IMenu, actionBar: ActionBar): boolean {
+	const inlineMenuItemActions = menu.getActions().find(a => a[0] === 'inline');
+	if (!inlineMenuItemActions || inlineMenuItemActions[1].length === 0) {
+		return false;
+	}
+
+	const actionBarMenuItemActions = actionBar.viewItems.map(a => a.action);
+	return inlineMenuItemActions[1].every(action => actionBarMenuItemActions.some(a => compareActions(a, action)));
```

Comment:
Will this still work if `menu` and `actionBar` have the same actions in a different order? Maybe that doesn't matter though.

Reply:
At the moment the order is not take into account but I will check with @joaomoreno if that is needed. Thanks!